sharing-data.html

```diff
@@ -203,7 +289,9 @@
 <div class="line"><br /></div> <!-- end #line -->
 </div> <!-- end #line-block -->
 
-<p>Use the <span class="bold">Display Users</span> checkbox to see the group's data irrespective of owner.</p>
+</li>
+
+<li><p>Use the <span class="bold">Display Users</span> checkbox to see the group's data irrespective of owner.</p>
```

Comment:
Maybe say: "By default, data is shown according to the owner of the data. Uncheck the "Display Users" checkbox to see all data from the group combined.